src/elastio-snap.c

```diff
@@ -6033,7 +6068,8 @@ static int elastio_snap_proc_show(struct seq_file *m, void *v){
 		if(error) seq_printf(m, "\t\t\t\"error\": %d,\n", error);
 
 		seq_printf(m, "\t\t\t\"state\": %lu,\n", dev->sd_state);
-		seq_printf(m, "\t\t\t\"ignore_errors\": %i\n", dev->sd_ignore_snap_errors);
+		seq_printf(m, "\t\t\t\"ignore_errors\": %i,\n", dev->sd_ignore_snap_errors);
+		seq_printf(m, "\t\t\t\"cow_on_bdev\": %s\n", test_bit(COW_ON_BDEV, &dev->sd_cow_state) ? "true" : "false");
```

Comment:
One more small "API change" for us ( We are using `info` IOCTL pretty actively. It's Ok. Just noticed )